We disable BSD usb joystick support until upstream has a better support for it
so that it can at least build. For now it builds but does not link to libusbhid
so that libglut.so has undefined references to hid_* symbols and causes linking
errors later on in the build process.

--- freeglut-2.4.0/src/freeglut_joystick.c.old	2008-08-08 15:26:15 +0000
+++ freeglut-2.4.0/src/freeglut_joystick.c	2008-08-08 15:26:32 +0000
@@ -78,7 +78,7 @@
 #    include <errno.h>
 #    if defined(__FreeBSD__) || defined(__NetBSD__)
 /* XXX The below hack is done until freeglut's autoconf is updated. */
-#        define HAVE_USB_JS    1
+#        undef HAVE_USB_JS
 
 #        if defined(__FreeBSD__) && __FreeBSD_version >= 500000
 #            include <sys/joystick.h>
